Jenkinsfile

```diff
@@ -0,0 +1,63 @@
+library changelog: false, identifier: 'lib@master', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/Percona-Lab/jenkins-pipelines.git'
+]) _
+
+pipeline {
+    agent {
+        label 'large-amazon'
+    }
+    parameters {
+        choice(
+            choices: [ 'test', 'e2e' ],
+            description: 'Run UI tests?',
+            name: 'RUN_TEST')
+        string(
+            defaultValue: '80.0',
```

Comment:
okay, this is good, but we also need to make it a choice, to avoid confusion from user input and only show supported values.